### ISSUE TITLE
Im having troble loading the aws toolkit for visual studio 2019 and 2017

### ISSUE THREAD

**Issue by joshg615** [aws/aws-toolkit-visual-studio#9]
I checked the activity log and this is the error I saw.

SetSite failed for package [AWSToolkitPackage]Source: 'EnvDTE' Description: Error HRESULT E_FAIL has been returned from a call to a COM component. System.Runtime.InteropServices.COMException (0x80004005): Error HRESULT E_FAIL has been returned from a call to a COM component.   at EnvDTE.Events.get_DebuggerEvents()   at Amazon.AWSToolkit.VisualStudio.AWSToolkitPackage.<InitializeAsync>d__67.MoveNext() --- End of stack trace from previous location where exception was thrown ---    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)   at Microsoft.VisualStudio.Shell.AsyncPackage.<>c__DisplayClass19_0.<<Microsoft-VisualStudio-Shell-Interop-IAsyncLoadablePackageInitialize-Initialize>b__1>d.MoveNext() --- End of stack trace from previous location where exception was thrown ---    at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)   at Microsoft.VisualStudio.Threading.JoinableTask.<JoinAsync>d__68.MoveNext() --- End of stack trace from previous location where exception was thrown ---    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()   at Microsoft.VisualStudio.Services.VsTask.RethrowException(AggregateException e)   at Microsoft.VisualStudio.Services.VsTask.InternalGetResult(Boolean ignoreUIThreadCheck)   at Microsoft.VisualStudio.Services.VsTask.GetResult()
--

**Comment by poojamandava** [comment]
Hi, is there any update on the issue, I'm getting the same error when trying to open AWS Explorer 

Open VS 2019 > Added AWS Toolkit for Visual Studio 2019 v 1.1.5.2.1 >  open AWS Explorer >getting below error:

SetSite failed for package [AWSToolkitPackage]Source: 'EnvDTE' Description: Error HRESULT E_FAIL has been returned from a call to a COM component. System.Runtime.InteropServices.COMException (0x80004005): Error HRESULT E_FAIL has been returned from a call to a COM component. at EnvDTE.Events.get_DebuggerEvents() at Amazon.AWSToolkit.VisualStudio.AWSToolkitPackage.<>c__DisplayClass67_1.<<InitializeAsync>b__0>d.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at Microsoft.VisualStudio.Threading.JoinableTask.<JoinAsync>d__68.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at Amazon.AWSToolkit.VisualStudio.AWSToolkitPackage.<InitializeAsync>d__67.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at Microsoft.VisualStudio.Shell.AsyncPackage.<>c__DisplayClass19_0.<<Microsoft-VisualStudio-Shell-Interop-IAsyncLoadablePackageInitialize-Initialize>b__1>d.MoveNext() --- End of stack trace from previous location where exception was thrown --- at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) at Microsoft.VisualStudio.Shell.UIThreadReentrancyScope.WaitOnTaskCompleteInternal(Task task, CancellationToken cancel, Int32 ms) at Microsoft.VisualStudio.Services.VsTask.<>c__DisplayClass100_1.<InternalGetResult>b__1() at Microsoft.VisualStudio.Services.VsTask.IgnoreObjectDisposedException(Action action) at Microsoft.VisualStudio.Services.VsTask.<>c__DisplayClass100_0.<InternalGetResult>b__0() at Microsoft.VisualStudio.Services.VsTask.<>c__DisplayClass108_0.<InvokeWithWaitDialog>b__0() at Microsoft.VisualStudio.Services.VsTask.InvokeWithWaitDialog[T](Func`1 function) at Microsoft.VisualStudio.Services.VsTask.InvokeWithWaitDialog(Action action) at Microsoft.VisualStudio.Services.VsTask.InternalGetResult(Boolean ignoreUIThreadCheck) at Microsoft.VisualStudio.Services.VsTask.GetResult()

**Comment by sambains** [comment]
I am also still getting this issue. Fresh install of Windows 10, Visual Studio 2019 Community and the AWS Toolkit v1.15.2.1.

The log is:
<entry>
    <record>182</record>
    <time>2019/11/13 22:18:41.968</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>SetSite failed for package [AWSToolkitPackage]Source: &apos;EnvDTE&apos; Description: Error HRESULT E_FAIL has been returned from a call to a COM component.&#x000D;&#x000A;System.Runtime.InteropServices.COMException (0x80004005): Error HRESULT E_FAIL has been returned from a call to a COM component.&#x000D;&#x000A;   at EnvDTE.Events.get_DebuggerEvents()&#x000D;&#x000A;   at Amazon.AWSToolkit.VisualStudio.AWSToolkitPackage.&lt;&gt;c__DisplayClass67_1.&lt;&lt;InitializeAsync&gt;b__0&gt;d.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#x000D;&#x000A;   at Microsoft.VisualStudio.Threading.JoinableTask.&lt;JoinAsync&gt;d__68.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#x000D;&#x000A;   at Amazon.AWSToolkit.VisualStudio.AWSToolkitPackage.&lt;InitializeAsync&gt;d__67.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.AsyncPackage.&lt;&gt;c__DisplayClass19_0.&lt;&lt;Microsoft-VisualStudio-Shell-Interop-IAsyncLoadablePackageInitialize-Initialize&gt;b__1&gt;d.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#x000D;&#x000A;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#x000D;&#x000A;   at Microsoft.VisualStudio.Threading.JoinableTask.&lt;JoinAsync&gt;d__68.MoveNext()&#x000D;&#x000A;--- End of stack trace from previous location where exception was thrown ---&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.RethrowException(AggregateException e)&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.InternalGetResult(Boolean ignoreUIThreadCheck)&#x000D;&#x000A;   at Microsoft.VisualStudio.Services.VsTask.GetResult()</description>
    <guid>{12ED248B-6D4A-47EB-BE9E-8EABEA0FF119}</guid>
    <hr>80004005 - E_FAIL</hr>
    <errorinfo></errorinfo>
  </entry>
  <entry>
    <record>183</record>
    <time>2019/11/13 22:18:42.982</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>End package load [AWSToolkitPackage]</description>
    <guid>{12ED248B-6D4A-47EB-BE9E-8EABEA0FF119}</guid>
    <hr>80004005 - E_FAIL</hr>
    <errorinfo></errorinfo>
  </entry>

**Comment by awschristou** [comment]
This is the same issue as #15 -- A fix is queued up for situations where the ActivityLog contains the following contents (specifically, "EnvDTE" and "get_DebuggerEvents"):

```
Source: &apos;EnvDTE&apos; Description: Error HRESULT E_FAIL has been returned from a call to a COM component.&#x000D;&#x000A;System.Runtime.InteropServices.COMException (0x80004005): Error HRESULT E_FAIL has been returned from a call to a COM component.&#x000D;&#x000A;   at EnvDTE.Events.get_DebuggerEvents()&#x000D;&#x000A;   at Amazon.AWSToolkit.VisualStudio.AWSToolkitPackage.Initialize()&#x000D;&#x000A;
```

As a workaround, you can install a debugger into Visual Studio as follows:
* Launch the Visual Studio Installer
* Modify your VS installation
* Click the Individual components tab
* Under the "Development activities" section, check "C# and Visual Basic"
* Install the checked components

**Comment by awschristou** [comment]
A fix for this is now available in v1.17.1.0 of the toolkit
